code/firmware/rosco_m68k_firmware/stage2/boot_menu/main.c

```diff
@@ -97,7 +97,11 @@ int main(void) {
         mcReadchar();
     }
 
-    do_boot_menu(demo_menu_items, demo_n_menu_items);
+    int res = do_boot_menu(demo_menu_items, demo_n_menu_items);
+
+    puts("\033c");
```

Comment:
Should these prints still be in here?

(FWIW I'm a bit surprised it actually builds, I thought pulling in printf and friends here would have blown up the size of the binary 🤷 ).

Reply:
Those were only for standalone build testing (hence why it builds)...I believe.  They could be removed.

Reply:
Ah okay, of course, makes sense :) No probs with keeping them then :)